applications/nvidia_nim/nvidia_nim_chat/README.md

```diff
@@ -0,0 +1,100 @@
+# Chat with NVIDIA NIM
+
+This is a sample application that shows how to use the OpenAI SDK with NVIDIA Inference Microservice (NIM). Whether you are using a NIM from [build.nvidia.com/](https://build.nvidia.com/) or a self-hosted NIM, this sample application will work for both.
+
+## Configuring the sample application
+
+Use the `nvidia_nim.yaml` configuration file to configure the sample application:
+
+### Connection Information
+
+```
+nim:
+  base_url: https://integrate.api.nvidia.com/v1
+  api_key:
+
+```
+
+`base_url`: The URL of your NIM instance. Defaults to NVIDIA hosted NIMs.
+`api_key`: Your API key to access NVIDIA hosted NIMs.
+
+### Model Information
+
+The `models` section int the YAML file is configured with multiple NVIDIA hosted models by default. This allows you to switch between different models easily within the application by sending `/m` to the application.
+
+Model parameters may be added or adjusted in the `models` section as well per model.
+
+## Run the sample application
+
+There are a couple of options to run the sample application:
+
+### Run using Docker
```

Comment:
Suggest "quick start" instructions with `dev_container build_and_run`:
1. Add API key in `nvidia_nim.yaml`
2. `./dev_container build_and_run nvidia_nim_chat`

Expected result is that the Dockerfile is detected, container builds, app builds, and app launches automatically with one command for streamlined user experience.

Reply:
Added Quick Start to the top of the README. Thank you!